sdk/trts/linux/trts_pic.h

```diff
@@ -39,6 +39,7 @@
 
 #include "linux/linux-regs.h"
 #include "rts_cmd.h"
+#include "trts_common_macro.h"
```

Comment:
"common_macro" is a non-suggestive name. Do you have plans to grow it outside of stack constants? Maybe trts_stack_constants.h would be better here.

Reply:
Actually we adding this header file is to put the macros shared between assembly code and C/C++ code put into one header file to avoid defining twice.  trts_stack_constants.h may not be good because we may have other shared constants which is not for stack.

Reply:
I update the patch to refine the comments and rename the header file as 'trts_shared_constants.h'.